vector/src/androidTest/java/im/vector/app/ui/robot/ElementRobot.kt

```diff
@@ -182,13 +181,8 @@ class ElementRobot {
             val activity = EspressoHelper.getCurrentActivity()!!
             val popup = activity.findViewById<View>(com.tapadoo.alerter.R.id.llAlertBackground)!!
             activity.runOnUiThread { popup.performClick() }
-
-            waitUntilViewVisible(withId(R.id.bottomSheetFragmentContainer))
```

Comment:
Unrelated to the FTUE change - the verification popup no longer features a skip button but can be exit via the back button

| BANNER | BOTTOMSHEET | 
| --- | --- | 
|![Screenshot_20220727_120404](https://user-images.githubusercontent.com/1848238/181247137-c54f69d5-9b91-4517-928c-6010cc744599.png)|![Screenshot_20220727_120410](https://user-images.githubusercontent.com/1848238/181247149-ccbb2cff-4fb8-4cd3-b629-6d2be130f5a4.png)